recipe_scrapers/amazingoriental.py

```diff
@@ -0,0 +1,30 @@
+from ._abstract import AbstractScraper
+
+
+class AmazingOriental(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "amazingoriental.com"
+
+    def title(self):
+        return self.soup.find('h1').get_text()
+
+    def category(self):
+        return self.soup.select("div.recipe-info .meal-type")[0].get_text()
+
+    def total_time(self):
+        return self.soup.select("dl.prepare-time dd")[0].get_text()
+
+    def yields(self):
+        return self.soup.select(".person-amount")[0].get_text()
```

Comment:
```suggestion
        return get_yields(self.soup.select(".person-amount")[0].get_text())
```
Just like the adjustment to time we have a similar helper function, `get yields`, to help normalize this output.